### ISSUE TITLE
[DLB]2018-01-20 질문사항

### ISSUE THREAD

**Issue by projokim** [YBIGTA/data-science-2018#3]
[1. Welcom to the Deep Learning.md](https://github.com/YBIGTA/DLB/blob/master/Neural%20Networks%20and%20Deep%20Learning/1.%20Welcom%20to%20the%20Deep%20Learning.md)

**Comment by statkim7578** [comment]
![22](https://user-images.githubusercontent.com/34841210/35153385-3ed90704-fd1e-11e7-970c-a97a58865931.PNG)

다중공선성 에서요
"두 개 이상의 설명변수들이 서로 밀접한 '상관'이 있는 경우를 말한다" 라고 하셨는데,
이 '상관'이라는게 두 변수간의 '직선관계' 만을 의미하는것인가요?
  
어떤 두 개의 설명변수들이 완벽한 '비선형적'인 관계를 갖고있다면, 다중공선성의 문제는 없는건가요?

**Comment by statkim7578** [comment]
![11](https://user-images.githubusercontent.com/34841210/35153782-d26a475c-fd1f-11e7-9cee-0b0473990dab.PNG)

'실제 설계에서 Neural Network 의 중간 레이어는 의미를 가지지 않는다' 라는 말은 왜 그런것인가요?

**Comment by Jooong** [comment]
sigmoid와 vanishing gradient 문제를 수식으로 풀어놓은 글은 [여기](https://brunch.co.kr/@chris-song/39)에서, 구현단에서 sigmoid와 relu의 성능 / gradient 크기 차이는 [여기](https://cs224d.stanford.edu/notebooks/vanishing_grad_example.html)에서 확인 가능합니다.

**Comment by KangBK0120** [comment]
@statkim7578 

저기서는 집의 크기와 침실의 수가 합쳐서 가족 구성원의 수와 높은 상관관계의 변수를 만들어내지만 

실제에서는 두 입력이 합쳐져도 무슨 관계인지 알 수 없다는 그런 의미인 것 같습니다. 이미지의 두 픽셀이 입력으로 들어간다음 나오는 결과가 무슨 의미를 가지는지는 알 수 없으니까요

**Comment by godongyoung** [comment]
<img width="631" alt="question2" src="https://user-images.githubusercontent.com/31824102/35162635-e840c9c2-fd3b-11e7-8439-d0961a90e949.PNG">
초반부의 집값 예측 부분에 대한 질문입니다. 
1. activation function이 '뉴런'의 기준이라고 이해하면 되는 건가요?
2. 만약 그래프가 ReLU형태가 아닌 단순히 y=x의 형태일때는 어떤 activation function이 들어가게 되나요?

**Comment by ybigtaJH** [comment]
1. 기존의 Sigmoid 함수가 1개이상의 Hidden layer을 넣을 수 없는 이유는 무엇인가요?
2. Neural Network가 structured data를 사용할 때, regression과 같은 기존의 방식에 비해 우수한 산출값이 나타나는 이유는, '더 많은 데이터의 입력에 따른 성능의 향상'뿐인가요?? 다른 이유는 없나요?

**Comment by jungsooyun** [comment]
![default](https://user-images.githubusercontent.com/32008845/35179047-3314169a-fd8a-11e7-9723-571bd91414b4.JPG)
예전에 프로젝트를 진행할 때, 분류모델에서 히든 레이어 수가 적은 경우가 많은 경우보다 성능이 좋게 나왔던 것으로 기억하는데 만약(?) 데이터의 수가 훨씬 방대해 진다면 히든 레이어 수가 많은 경우가 성능이 개선될 여지가 많다고 생각해도 되나요?

**Comment by hyejukim-ybigta** [comment]
large NN이 항상 small NN보다 성능이 좋나요? overfitting 문제는 어떻게 해결하나요?

**Comment by angrypark** [comment]
structured data의 경우, 아직까지도 neural network보단 forest 모델이 더 우위를 보이고 있는데, data의 크기와 차원, 풀고자 하는 문제에 따라 어떤 model을 도전할 것인지 선택하는 기준이 있을까요?

**Comment by DabinJeong** [comment]
뉴럴 네트워크가 구조화된 데이터 처리에 기반하고 있다면.. 이미지와 같은 구조화되지 않은 데이터도 구조화된 데이터로 바꾸어서 처리해야 한다는 말인가요??

**Comment by SongChiYoung** [comment]
testtest

**Comment by SongChiYoung** [comment]
Duplicate of #5 
safd

**Comment by davinnovation** [comment]
@Rafaela94 
"그럼에도 불구하고, 뉴런네트워크가 만들어내는 단기간의 경제적 가치는 '우수한 광고 시스템', '더 나은 수익모델' 등 거대 데이터베이스 처리 능력(구조화 된 데이터)에 기반하고 있다는 사실을 잊어서는 안됩니다."

이 문장을 오독하신 것 같습니다. NN은 structed, unstructed 상관이 없습니다. ( 사실 대부분의 ml 모델이 그렇죠 )
